ray-operator/apis/ray/v1/raycronjob_types.go

```diff
@@ -13,6 +13,10 @@ type RayCronJobSpec struct {
 	JobTemplate RayJobSpec `json:"jobTemplate"`
 	// Schedule is the cron schedule string
 	Schedule string `json:"schedule"`
+	// Suspend tells the controller to suspend the scheduling, it does not apply to
+	// scheduled RayJob.
+	// +optional
+	Suspend bool `json:"suspend,omitempty"`
```

Comment:
```suggestion
	Suspend *bool `json:"suspend,omitempty"`
```
I prefer using a pointer type for an optional boolean type.